cypress/cypress.d.ts

```diff
@@ -16,14 +16,14 @@
 
 import { mount } from 'cypress/react';
 
-// Augment the Cypress namespace to include type definitions for
+// Augment the Cypress name-space to include type definitions for
```

Comment:
Not sure if we just need to add `namespace` to the dictionary but this shouldn't need a hyphen.

Reply:
Adding it to dictionary is better. Ill do that